lbz/resource.py

```diff
@@ -101,24 +85,54 @@ def _get_user(self, headers: CIMultiDict) -> Union[None, User]:
         return None
 
     def _post_request_hook(self) -> None:
-        """
+        """*/
```

Comment:
?

Reply:
![Screenshot 2022-09-03 at 08 52 35](https://user-images.githubusercontent.com/9288974/188259468-5747b434-0227-4715-a86d-9234f7f56018.png)

Reply:
The original comment is about the protected method, not the public one - still valid 😉